app/routes/error.js

```diff
@@ -0,0 +1,3 @@
+import Route from '@ember/routing/route';
```

Comment:
Do we need this file if its empty?
If I remember correctly Ember will generate an empty route anyways no?

Reply:
Yep this can be 🔪 🎉